__tests__/app.test.js

```diff
@@ -104,18 +115,45 @@ describe("GET /api/reviews", () => {
         votes: 5,
         designer: "Leslie Scott",
         comment_count: 3,
-      }
+      },
     ];
-      return request(app)
-      .get('/api/reviews')
+    return request(app)
+      .get("/api/reviews")
       .then((result) => {
         expect(result.body.reviews).toEqual(expect.arrayContaining(expected));
-      })
+      });
   });
 });
 
 describe("GET /api/reviews/:review_id", () => {
   test("returns status code 200", () => {
+    return request(app).get("/api/reviews/1").expect(200);
+  });
+  test("responds with an object", () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .then(({ body }) => {
+        expect(typeof body.review).toBe("object");
+      });
+  });
+  test("responds with object with appropriate properties", () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .then(({ body }) => {
+        expect(body.review).toMatchObject({
+          review_id: expect.any(Number),
+          title: expect.any(String),
+          designer: expect.any(String),
+          owner: expect.any(String),
+          review_img_url: expect.any(String),
+          review_body: expect.any(String),
+          category: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+        });
+      });
+  });
+  test("responds with 404 upon being a number out of range", () => {
```

Comment:
This test does not test for a 404, it expects a 200 back

Reply:
I think you can probably delete it. It looks you have done sad path testing later on for this endpoint